src/pages/Delivery/Delivery.styled.jsx

```diff
@@ -6,55 +6,85 @@ const MapLayoutContainer = styled.div`
   height: 100vh;
   width: 100vw;
 `
-
 const HeaderMap = styled.div`
   display: flex;
   align-items: center;
   flex-direction: row;
-  background: var(--primary);
+  background: var(--blueheader);
   border-bottom-right-radius: 25px;
   border-bottom-left-radius: 25px;
-  padding: 5px;
-  min-height: 10vh;
+  padding: 3px;
+  min-height: 15vh;
   place-content: space-between;
+
   @media (max-width: 960px) {
-    width: inherit;
+    width: 99.3%;
     position: absolute;
     z-index: 1;
   }
+  @media (max-width: 885px) {
+    width: 99.1%;
+  }
+  @media (max-width: 767px) {
+    width: 99%;
+  }
+  @media (max-width: 677px) {
+    width: 98.9%;
+  }
+  @media (max-width: 606px) {
+    width: 98.3;
+  }
+  @media (max-width: 523px) {
+    width: 98.7%;
+  }
+  @media (max-width: 462px) {
+    width: 98.6%;
+  }
+  @media (max-width: 411px) {
+    width: 98.2%;
+  }
+  @media (max-width: 371px) {
+    width: 98.3%;
+  }
+  @media (max-width: 320px) {
+    width: 97.8%;
+  }
+  @media (max-width: 280px) {
+    width: 97.2%;
+  }
```

Comment:
What does this lines? I would like to help you to find a better solution for this problem